src/main/java/seedu/address/logic/Logic.java

```diff
@@ -51,6 +51,9 @@ public interface Logic {
     /** Returns an unmodifiable view of the filtered list of tasks */
     ObservableList<Task> getFilteredTaskList();
 
+    /** Returns an unmodifiable view of the tasks due soon */
+    ObservableList<Task> getTaskDueSoonList();
```

Comment:
should be `getTasksDueSoonList` to be more accurate

Reply:
Good catch! Changed it.